### PR TITLE
Css three column layout - Sina Daryaei 

### PR DESCRIPTION
**Pull request by sinadaryaei**
image

### CODE REVIEW

File: css-three-column-layout/index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  <title>CSS Challenge - Three Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <h1>Design for life science. Optimized for SaMD and Digital Health.</h1>
+  <div class="container">
+    <header class="row">
+    </header>
```
Comment (shawnkost) [inline css-three-column-layout/index.html:16]: What's the point of this, if you don't have any text inside of it? Should the `h1` on line 13 go inside here?

File: css-three-column-layout/styles.css [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+.column {
+  flex-basis: 50%;
+  padding: 10px;
+}
+
+.image {
+  max-width: 50%;
+  max-height: 50%;
+  padding: 20px;
+  width: 80%;
```
Comment (shawnkost) [inline css-three-column-layout/styles.css:27]: You set the `width` to 80%, but then you set the `max-width` to 50%. You shouldn't set a `width` that is bigger than your `max-width`.

File: css-three-column-layout/styles.css [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  font-family: lato, sans-serif;
+}
+
+.row {
+  display: flex;
+  width: 100%;
+}
+
+.column {
+  flex-basis: 50%;
```
Comment (shawnkost) [inline css-three-column-layout/styles.css:19]: This shouldn't be 50%. If you have 3 columns, you cant have all of them take up 50% right? You should do `flex-basis: calc(100% / 3)`

File: css-three-column-layout/styles.css [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+}
+
+.character-description {
+  font-size: normal;
+  text-align: center;
+}
+
+.title {
+  text-align: center;
+  font-size: 150%;
```
Comment (shawnkost) [inline css-three-column-layout/styles.css:48]: I don't recommend using percents for font-size. You should use either `rem` or `px`.

**Review by shawnkost — CHANGES_REQUESTED** [review]
Some changes requested here Sina!

**Comment by sinadaryaei** [comment]
Thank you, I'll make those changes.

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**